big-data-europe/docker-hadoop
# Replace keyserver key retrieval with key file download

Keyserver causes random errors (https://github.com/big-data-europe/docker-hadoop/issues/29). Use verification steps proposed by Hadoop instead:

```
To verify Hadoop releases using GPG:

    1. Download the release hadoop-X.Y.Z-src.tar.gz from a mirror site.
    2. Download the signature file hadoop-X.Y.Z-src.tar.gz.asc from Apache.
    3. Download the Hadoop KEYS file.
    4. gpg –import KEYS
    5. gpg –verify hadoop-X.Y.Z-src.tar.gz.asc
```

 source: https://hadoop.apache.org/releases.html
